migration/v4/model/jobfunction.go

```diff
@@ -0,0 +1,8 @@
+package model
+
+type JobFunction struct {
```

Comment:
hint: necessary to bring this forward since it has a relationship to a model that was updated in this migration (Application)